Makefile

```diff
@@ -0,0 +1,68 @@
+# HCA HOST used in the matrix service (Feel free to change it if necessary)
+HCA_HOST := https://dss.dev.data.humancellatlas.org/v1
+
+default: all
+
+.PHONY: install
+install:
+	virtualenv -p python3 venv
+	. venv/bin/activate && pip install -r requirements-dev.txt --upgrade
+
+.PHONY: test
+test:
+	. venv/bin/activate && PYTHONPATH=chalice \
+		python -m unittest discover -s tests -p '*_tests.py'
+
+.PHONY: build
+build:
+	@read -p "Enter AWS_PROFILE: " aws_profile; \
```

Comment:
consider using AWS secrets manager, and putting all of this config in a script that someone can run once.

Reply:
@l5han you can use this code https://github.com/HumanCellAtlas/dcplib/blob/master/dcplib/aws_secret.py